Classes/Neos/Neos/Ui/Aspects/AugmentationAspect.php

```diff
@@ -155,7 +155,7 @@ public function contentElementAugmentation(JoinPointInterface $joinPoint)
         $this->userLocaleService->switchToUILocale(true);
 
         $wrappedContent = $this->htmlAugmenter->addAttributes($content, $attributes, 'div');
-        $wrappedContent .= "<script>(function(){(this['@Neos.Neos.Ui:Nodes'] = this['@Neos.Neos.Ui:Nodes'] || {})['{$node->getContextPath()}'] = {$serializedNode}})()</script>";
+        $wrappedContent .= "<script data-neos-inlinescript='true'>(function(){(this['@Neos.Neos.Ui:Nodes'] = this['@Neos.Neos.Ui:Nodes'] || {})['{$node->getContextPath()}'] = {$serializedNode}})()</script>";
```

Comment:
we forgot to test, is `='true'` needed here. Probably not.

Reply:
And yes, naming. This is not just inline script. It's more like `data-neos-nodedata`.